docker-compose.yml

```
@@ -22,7 +21,17 @@ services:
       args:
         - RUN_UID=${RUN_UID:-4000}
         - RUN_USER=${RUN_USER:-app}
-    command: ["poetry", "run", "chainlit", "run", "src/app.py", "-w"]
+    command:
+      [
+        "poetry",
+        "run",
+        "uvicorn",
+        "src.app:app",
+        "--host",
+        "0.0.0.0",
+        "--port",
+        "8000",
+      ]
```

Since the docker compose command is used for local development, we can support hot reloading when the code changes with:
```suggestion
        "8000",
        "--reload",
      ]
```

---

Ah, one more note here -- can you also aupdate the `app/Dockerfile` commands (there are two in the file), which are currently `run chainlit`, to use this command instead?